src/mame/drivers/xexex.cpp

```diff
@@ -128,6 +128,8 @@ Unresolved Issues:
 - the stage 4 boss(tentacles) sometimes appears darker (palette update timing?)
 - the furthest layer in stage 5 shakes when scrolling up or down (needs verification)
 - Elaine's end-game graphics has wrong masking effect (known non-zoomed pdrawgfx issue)
+- intro screen at wrong alpha effect; it has K054157 flag-per-tile for alpha blending?
```

Comment:
No, it's keyed on the palette.  Don't spend time on the graphics issues though, it's worked on in another branch.